about.html

```diff
@@ -4,61 +4,85 @@
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
+        <div class="sectiongrid">
+          <section>
+            <img src="img/about-plan.png" alt="strategy">
+            <h2>Strategy</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <a href="">Learn More</a>
+          </section>
+          <section>
+              <img src="img/about-working.png" alt="strategy">
+            <h2>How We Work</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <a href="">Learn More</a>
+          </section>
+          <section>
+            <img src="img/about-office.png" alt="strategy">
+            <h2>Places We Work</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <a href="">Learn More</a>
+          </section>
+          <section>
+            <img src="img/about-meeting.png" alt="strategy">
+            <h2>Collaboration</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <a href="">Learn More</a>
+          </section>
+        </div>
+        
+        <summary>
+          <h2>Let's Work Together</h2>
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+        </summary>
+        
+        
+        
+        
+        
     
-    How We Work
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
+        
     
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            
 
-    Learn More
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
 
-    Let's Work Together
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+      </main>
+              
       <footer>
         <nav>
```

Comment:
Great work on giving semantic meaning to your HTML and good job on making use of comments to each semantic element.